app/components/ui/tale-tab-files/component.js

```diff
@@ -0,0 +1,12 @@
+import Component from '@ember/component';
+
+
+export default Component.extend({
+  model: null,
+  init() {
+    this._super(...arguments);
+  },
+  
+  actions: {
```

Comment:
If no actions performed by this component, I'd rather not include the `actions` hook. This is not a mistake, just more lines for the compiler to process, which is already slow as it is.

Reply:
This was a stub, as we will obviously be adding functionality to this page in the near future.